src/dsp_tools/commands/start_stack.py

```diff
@@ -293,11 +286,17 @@ def _start_remaining_docker_containers(self) -> None:
         (Fuseki is already running at this point.)
         """
         if self.__stack_configuration.latest_dev_version:
-            subprocess.run(
-                "docker pull daschswiss/knora-api:latest".split(),
-                cwd=self.__docker_path_of_user,
-                check=True,
+            override_file = importlib.resources.files("dsp_tools").joinpath(
+                "resources/start-stack/docker-compose.override.yml"
             )
+            override_content = yaml.safe_load(override_file.read_bytes())
+            dockerhub_latest_tags = [x["image"] for x in override_content["services"].values()]
+            for tag in dockerhub_latest_tags:
+                subprocess.run(
+                    f"docker pull {tag}".split(),
+                    cwd=self.__docker_path_of_user,
+                    check=True,
+                )
```

Comment:
This is the actual change of this PR. This is intentionally coupled to `docker-compose.override.yml`, because 2 things are necessary in order to get the latest tag from a dockerhub-image:
1. The image needs to be pulled (this happens here)
2. In the `docker-compose.yml`, the "latest" tag must be present (this is done by `docker-compose.override.yml`)